doc/contributor/howto-guide-generated-code-maintenance.md

```diff
@@ -15,6 +15,35 @@ fork:
 git checkout -b feat-websecurityscanner-generate-library
 ```
 
+Generate the library:
+
+```bash
+go -C generator/ run ./cmd/sidekick rust-generate \
+    -project-root .. \
+    -service-config google/cloud/websecurityscanner/v1/websecurityscanner_v1.yaml
+```
+
+Compile and run the tests:
+
+```bash
+typos && cargo build && cargo test && cargo doc
```

Comment:
This is done for the runner now, can be removed.

Reply:
What is `runner` ?  Sidekick does not run the tests or builds the code.

Reply:
oops you are right!